### ISSUE TITLE
RPG not working in standard Build & Shoot client

### ISSUE THREAD

**Issue by SmokeysDev** [Night-Stalkers/ns-scripts#11]
Server: Battle Bots TDM
Client: Build & Shoot

It seems a tad buggy but mostly fine in OpenSpades, but playing in Build & Shoot client today I couldn't for the life of me get the RPG to fire, just kept throwing grenades

**Comment by hourai-dev** [comment]
Can you try reproducing it again using the latest version of rpg_mod?

**Comment by SmokeysDev** [comment]
**TLDR:** Bug isn't in Build & Shoot, it's in OpenSpades. I was firing RPGs wrong this whole time.

Still an issue.. or so I thought. After adding in some debug statements and re-reading the code it looks like I've been firing RPGs wrong. In OpenSpades you can hold both mouse buttons and then release the left mouse button pretty much immediately to fire the RPG quickly. That doesn't work in Build & Shoot, which is what I was bugging - what does work on both clients, is holding both buttons down until eventually the RPG fires of its own accord. So, assuming that's the intended method, we should maybe close this and instead open a bug for how it works on OpenSpades?
